### DISCUSSION TITLE
How to make ripgrep not match filenames?

### DISCUSSION THREAD

**Discussion by RakshithNM** [BurntSushi/ripgrep#1880] [General] [answered]
Lets say i have a file `canvas.ts` and i am searching for `canvas` pattern. It matches the the filename `canvas.ts`  at this point(see image for better clarity)
![Screenshot 2021-05-30 at 12 08 31 PM](https://user-images.githubusercontent.com/10560321/120094726-c230a380-c13f-11eb-9f1a-c71160b3e683.png)

**Comment by BurntSushi** [comment, marked as answer]
You aren't running ripgrep though. Or at least, you're not running _only_ ripgrep. You're running some other tool that is either using ripgrep to match filenames or is matching filenames itself.

When asking questions about ripgrep, I definitely prefer that you do it in the context of running `rg` commands on the command line. In that context, `rg pattern` only searches the contents of files for `pattern`. It does not also search filenames.

**Reply by RakshithNM** [reply]
yeah i am running it in fzf.vim, will ask the question there, thanks for clarifying
